pkg/framework/retry.go

```diff
@@ -1,35 +0,0 @@
-package framework
-
-import (
-	"fmt"
-	"log"
-	"time"
-
-	"github.com/cenkalti/backoff"
-)
-
-const maxElapsedTime = 2 * time.Minute
-
-// newCustomExponentialBackoff returns pointer to a backoff.ExponentialBackOff,
-// initialized with custom values. At the moment, we only override the
-// maxElapsedTime.
-func newCustomExponentialBackoff() *backoff.ExponentialBackOff {
```

Comment:
This is the same function as `NewExponentialBackoff` with one less input parameter. We used this once and can simply pass the default in the second value.